README.md

```diff
@@ -14,52 +14,114 @@ When you are done:
 
 #### Loops & Lists
 - [Max Value](https://repl.it/@Admin7/maxvalue)
-```python
-# copy & paste your solution here
+```
+def return_max(l):
+  
+  if len(l) == 0:
+    return None 
+  
+  max_number = l[0]
+  for i in range (1, len(l)):
+    if l[i] > max_number: 
+      max_number = l[i]
```

Comment:
Great job doing this while also practicing loops! This is exactly what I was looking for. Moving forward, you can also use the built in `min` and `max` functions!